src/SarifWeb/Views/ValidationUi/Index.cshtml

```diff
@@ -57,15 +57,19 @@
                 <div id="ruleDescription"></div>
             </div>
             <div id="rightToolbarGroup">
-                <button id="downloadLogButton" aria-label="Download log file" title="Download log file" class="toolbar-button" disabled>
```

Comment:
>downloadLogButton [](start = 28, length = 17)

No need for this because it was used to download the _transformed_ log, which we no longer produce.